### DISCUSSION TITLE
打包 ZerotierFix 的时候如何预置 planet 文件

### DISCUSSION THREAD

**Discussion by player2019** [kaaass/ZerotierFix#106] [Q&A]
大佬你好，我们这边在自行打包ZerotierFix的时候碰到问题无法解决，请问能有偿协助我们解决吗？已向大佬博客about页面中留下的邮箱发送了邮件，未能获得回复，所以在此留言，希望能看到，感激不尽

**Comment by kaaass** [comment]
请发一下具体的问题哈，这样如果有类似问题也可以帮助其他朋友。如果不太方便的话也可以向我的邮箱发送，我刚 check 了下已经收到你之前的邮件了。

**Reply by player2019** [reply]
一番折腾，app现在是打包出来了，但是源码中似乎没有找到 planet 文件，请教大佬该如何替换成自己的 palnet？

已修改标题
